src/main/io/osd.c

```diff
@@ -103,6 +103,11 @@
 #define VIDEO_BUFFER_CHARS_PAL    480
 #define FULL_CIRCLE 360
 
+#define STICK_OVERLAY_HORIZONTAL_CHAR   0x2D
```

Comment:
It may be cleaner to use character constants (`+`,`-`,`|`,`0`)

Reply:
Done.